packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -17,8 +19,19 @@ import {
   SObjectCategory,
   SObjectDescribe
 } from '../describe';
+import { nls } from '../messages';
+
+export interface CancellationToken {
+  isCancellationRequested: boolean;
+}
 
+export const SUCCESS_CODE = '0';
+export const FAILURE_CODE = '1';
+const SFDX_PROJECT_FILE = 'sfdx-project.json';
 export class FauxClassGenerator {
+  private emitter: EventEmitter;
+  private cancellationToken: CancellationToken | undefined;
+
   private SFDX_DIR = '.sfdx';
```

Comment:
Why can't these be constants as well? I think I asked this during the previous code review.

Reply:
could be and should be - I'll fix